articles/152_roslaunch_multi_machine_launch.md

```diff
@@ -0,0 +1,310 @@
+---
+layout: default
+title: ROS 2 Multi-Machine Launching
+permalink: articles/roslaunch_mml.html
+abstract:
+  Robotic systems are often distributed across multiple networked machines.
+  This document describes proposed modifications and enhancements to ROS2's
+  launch system to facilitate launching, monitoring, and shutting
+  down systems spread across multiple machines.
+author: '[Matt Lanting](https://github.com/mlanting)'
+published: false
+---
+
+- This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Authors: {{ page.author }}
+
+## Purpose
+
+Allow a system of ROS nodes to be launched on a hardware architecture that is
+spread across multiple networked computers and facilitate introspection and
+management of the system from a single machine.
+
+## Justification
+
+Nodes can need to run on different hosts for a variety of reasons.  Some possible
+use cases:
+
+- A large robot could with hosts located physically near the hardware they are
+controlling such as cameras or other sensors
+- A robot with hosts with different architectures in order to use specialized
+processing hardware
+- A robot with a cluster of machines that do distributed processing of data
+- A network of multiple virtual hosts for testing purposes
+- A swarm of independent drones that can cooperate but do not require
```

Comment:
```suggestion
- A swarm of cooperative, independent agents that do not require intercommunication
```

Reply:
Why limit to a swarm and not a team of more than 1 agent?

How is agent intercommunication relevant to multi-machine launching? And why limit to systems without it?

Why not just: Cooperative Multi-Agent Systems?

Reply:
It's just a specific use case example. I don't think it was meant in an all-encompassing manner for multi-machine launching.